src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.component.html

```diff
@@ -18,7 +13,7 @@ <h3>You can view your recent work items</h3>
           Planner will help you to create different types of work-items, allows you to assign team members and iterations.
         </p>
         <div class="f8-blank-slate-main-action">
-          <button id="spacehome-my-workitems-create-button" class="btn btn-primary btn-lg" [routerLink]="[{ outlets: { action: 'add-work-item' } }]">Create work item</button>
+          <button id="spacehome-my-workitems-create-button" class="btn btn-primary btn-lg" [routerLink]="[contextPath | async, 'plan']">View planner</button>
```

Comment:
@catrobson do we have guidelines on casing for button labels like this?

![screenshot from 2018-05-02 14-02-11](https://user-images.githubusercontent.com/3787464/39540717-7e98838a-4e11-11e8-8b09-04359d0156d4.png)

Here we have "View planner", but other places in the same view have "Add Codebase", "Add to Space" and "Add to space".